internal/pkg/scaffold/ansible/go_mod.go

```diff
@@ -109,6 +109,7 @@ replace (
 	github.com/coreos/prometheus-operator => github.com/coreos/prometheus-operator v0.29.0
 	k8s.io/code-generator => k8s.io/code-generator v0.0.0-20181117043124-c2090bec4d9b
 	k8s.io/kube-openapi => k8s.io/kube-openapi v0.0.0-20180711000925-0cf8f7e6ed1d
+	github.com/operator-framework/operator-sdk => github.com/operator-framework/operator-sdk v0.8.0
```

Comment:
Should we change the version in the `require` section or keep this as a separate `replace`? 

I read [this](https://github.com/golang/go/wiki/Modules#when-should-i-use-the-replace-directive), and I still don't think I know which is the correct way to do this. Thoughts?

/cc @estroz

Reply:
I just followed what we have in our release doc. I feel like the replace statement makes sense for this situation, but I don't know much about go mod

Reply:
`replace` pins a module version absolutely. This is similar to a `Gopkg.toml` override. See the last point in the replace section of [this doc](https://github.com/thepudds/go-module-knobs/blob/master/README.md).